gomock.bzl

```diff
@@ -6,23 +6,22 @@ _MOCKGEN_MODEL_LIB = "@com_github_golang_mock//mockgen/model:go_default_library"
 
 def _gomock_source_impl(ctx):
     go_ctx = go_context(ctx)
-    gopath = "$(pwd)/" + ctx.bin_dir.path + "/" + ctx.attr.gopath_dep[GoPath].gopath
 
     # passed in source needs to be in gopath to not trigger module mode
-    args = ["-source", gopath + "/src/" + ctx.attr.library[GoLibrary].importmap + "/"+ ctx.file.source.basename]
+    args = ["-source", ctx.file.source.path]
 
     args, needed_files = _handle_shared_args(ctx, args)
 
     if len(ctx.attr.aux_files) > 0:
         aux_files = []
-        for pkg, files in ctx.attr.aux_files.items():
-            for f in files:
-                mapped_f = gopath + "/src/" + ctx.attr.library[GoLibrary].importmap + "/"+ f
-                aux_files.append("{0}={1}".format(pkg, mapped_f))
+        for target, pkg in ctx.attr.aux_files.items():
+            f = target.files.to_list()[0]
+            aux_files.append("{0}={1}".format(pkg, f.path))
+            needed_files.append(f)
```

Comment:
This looks like an API break. Why are we doing it?

Reply:
Refer to my other comment

Reply:
Also, confirmed with @josmad, who contributed `aux_files` attribute, that he is not using the attribute anywhere right now

Reply:
@jmhodges Can you make a major version bump for this API break?